.github/workflows/trunk.yml

```diff
@@ -18,8 +18,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test-models-macos:
-    name: test-models-macos
+  test-models-macos-cpu:
+    name: test-models-macos-cpu
```

Comment:
Nit: Just to more backend specific (and not confuse with XNNPACK)
```suggestion
    name: test-models-macos-portable
```

Reply:
it's actually both... xnnpack and portable, lol